.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
     validation:
         name: "CS, ArchSniffer, PHPStan, Security"
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-latest
```

Comment:
We usually keep the version here to avoid breaking things once they release a new version.
We manually upgrade
Rest of the changes look fine